pyproject.toml

```diff
@@ -62,6 +62,7 @@ dev = [
     "types-requests",
     "uv",
     "pre-commit",
+    "cffconvert",
```

Comment:
not used anymore, right?

Reply:
I see, I do not use the python lib, we can skip it for now.